openslides_backend/action/relations/meeting_user_ids_handler.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict
```

Comment:
This code is meeting specific, right? Maybe it should not live in the generic "relation" directory.

Reply:
Mmh, it's not _really_ meeting specific since it's also called on fields from `group`. Also, I would not like to move it into the `actions` folder since it's not an action and not directly action-related. If you have a good suggestion where to put it, we can move it, else I would suggest to leave it here for now.